docs/kafka-messages.rst

```diff
@@ -177,4 +259,76 @@ configured with the following message:
 The attribute actually used by the device would be "target_temperature" so that
 it can adjust correctly the temperature. It's up to the receiver of this
 message (an IoT agent, for instance) to properly send the configuration to the
-device.
\ No newline at end of file
+device.
+
+
+Template update message
+-----------------------
+
+This message is published whenever a template gets updated. It contains all the
+affected devices and the new model for that template. Important thing to
+remember: no message is sent to update *each device*.
+
+Its payload looks like:
+
+.. code-block:: json
+
+  {
+    "event": "template.update",
+    "data": {
+        "affected": [
+            "9c6f77"
+        ],
+        "template": {
+            "label": "SuperTemplate",
+            "id": 1,
+            "created": "2019-01-07T12:03:47.051392+00:00",
+            "attrs": [
+                {
+                    "label": "a",
+                    "id": 3,
+                    "template_id": "1",
+                    "type": "dynamic",
+                    "created": "2019-01-07T12:03:47.055768+00:00",
+                    "value_type": "string"
+                },
+                {
+                    "label": "b",
+                    "id": 4,
+                    "template_id": "1",
+                    "type": "dynamic",
+                    "created": "2019-01-07T12:03:47.056419+00:00",
+                    "value_type": "string"
+                },
+                {
+                    "label": "c",
+                    "id": 6,
+                    "template_id": "1",
+                    "type": "dynamic",
+                    "created": "2019-01-07T12:11:42.971507+00:00",
+                    "value_type": "string"
+                }
+            ]
+        }
+    },
+    "meta": {
+        "service": "admin"
+    }
+  }
+
+
+- *event* (string): "template.update"
```

Comment:
Why the template related event has the "template." prefix and the device related events has not the prefix "device."?

Reply:
These events are not yet standardized. Maybe, who knows, in the future, when the sky is blue and fluffy bunnies dance to a soothing song the sun sings with a full double rainbow by its side, we could implement that.